core/modules/parser/SConscript

```diff
@@ -17,15 +17,19 @@ antlrTgts = env.Antlr(['SqlSQL2.g'],
                       ANTLR_OUT='.',
                       )
 
+antlr4LexerTgt = env.Antlr4(['MySqlLexer.g4'], ANTLR4_OUT='.')
+                      
+antlr4ParserTgt = env.Antlr4(['MySqlLexer.tokens', 'MySqlParser.g4'], ANTLR4_OUT='.')
```

Comment:
Maybe add a comment here that `MySqlLexer.tokens` is on the target list from the previous line?